.env.example

```
@@ -5,9 +5,100 @@ OLLAMA_BASE_URL='http://localhost:11434'
+# MS_GRAPH_HEADERS='X-MS-Token-AAD-Access-Token,X-MS-Token-AAD-Id-Token,X-Forwarded-AAD-Access-Token'
+
+# NewsDesk Azure Blob Storage configuration for MCP news server
+AZURE_STORAGE_CONNECTION_STRING=''
+AZURE_BLOB_CONTAINER_NAME='articles'
+
+# SharePoint MCP Integration Environment Variables
+# Uses Microsoft Graph API with On-Behalf-Of (OBO) flow for user-scoped access
+# Multi-department support - each department has its own set of credentials
+
+# ===== GLOBAL SHAREPOINT SETTINGS (shared across all departments) =====
+SHP_USE_DELEGATED_ACCESS=true # Enable user-scoped OBO access (recommended: true)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)